galaxykit/client.py

```diff
@@ -281,3 +277,45 @@ def delete_collection(self, namespace, collection, version, repository):
     def deprecate_collection(self, namespace, collection, repository):
         """deprecates a collection"""
         return collections.deprecate_collection(self, namespace, collection, repository)
+
```

Comment:
as we talked about, I don't think we strictly need all this boilerplate - bad design decision on my part when I first created galaxykit to attempt to wrap every module with a function in `client.py`.

Furthermore, if we wanted to do this type of thing down the line, it'd make more sense to do some type of metaprogramming (i.e. potentially with decorators or something) rather than to manually write all these functions.

But for now I say we can just remove these.